.github/workflows/run-test-harness.yml

```diff
@@ -10,8 +10,9 @@ jobs:
       labels: ubuntu-latest-4-core
     steps:
       - uses: DevCycleHQ/test-harness@main
+        env:
```

Comment:
local bucketing too - and all those

Reply:
copied over the test-harness capabilities, will add

Reply:
LB was added as a default i think